lib/rules/unit-disallowed-list/README.md

```diff
@@ -157,3 +157,27 @@ The following patterns are considered problems:
 ```css
 @media print and (max-resolution: 100dpi) {}
 ```
+
+### `ignoreFunctions: ["function", "/regex/", /regex/]|"function"|"/regex/"|/regex/`
+
+Ignore units that are inside of the specified functions.
+
+For example, with `["px"]`.
+
+Given:
+
+```json
+"ignoreFunctions": ["calc", "/^translate/"]
```

Comment:
[suggest] For JSON validity and consistency, can you please update it?

```suggestion
["calc", "/^translate/"]
```

Reply:
Thanks! I've applied the change; I noticed a similar pattern in `unit-allowed-list` which I based the documentation off of:

https://github.com/stylelint/stylelint/blob/b447409c2f7a619f663d7888e98050cba6a636d7/lib/rules/unit-allowed-list/README.md?plain=1#L125-L129

It seems inconsistent with the rest of the docs, so I'll also submit a (tiny) PR to resolve that. Great eye!

Reply:
(opened a micro-PR in https://github.com/stylelint/stylelint/pull/6594; let me know if that was the desired behaviour)